zqd/space/space.go

```diff
@@ -43,6 +45,14 @@ type Space interface {
 	update(api.SpacePutRequest) error
 }
 
+// PcapCapableSpace denotes that a space is capable of storing pcap files and
+// indexes.
+// XXX Temporary. The should be removed once archive spaces are enabled to allow
+// pcap ingest.
+type PcapCapableSpace interface {
+	PcapStorage() *pcapstorage.Store
+}
```

Comment:
```suggestion
type PcapSpace interface {
	PcapStore() *pcapstorage.Store
}
```

Reply:
I like `PcapStore`. The name `PcapSpace` was throwing me off. That said happy to change this back. This will only be temporary. Once I add pcap support to archive spaces `PcapStore()` will move to the general `Space` interface.